commit c587e86eea7fb51fc917b7d0ece816bcd52aafa7 (refs/changes/62/2062/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-06-08T15:08:44-07:00 (2 years, 4 months ago)
    
    MANATEE-333 manatee-adm could report sync replay lag
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/package.json b/package.json
index ff84678..bfd162e 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "carrier": "0.2.0",
-        "manatee": "git+https://github.com/joyent/manatee.git#86ae52d4a9988958aec3ec9955f3e2190bfa81c7"
+        "manatee": "git+https://github.com/joyent/manatee.git#5e4ab2c6cc25849cce78c35a4f5cef5a74a26a69"
     },
     "sdcDependencies": {
         "config-agent": ">=1.2.0"
